src/h5web/providers/mock/data.ts

```diff
@@ -368,4 +369,5 @@ export const mockValues = {
   fourD,
   X: arr1,
   Y: arr2,
+  title: 'NeXus 2D',
```

Comment:
Maybe use a more distinctive ID, like `title_twoD` and then call `makeDatasetHardLink` with a second argument?